code/src/Components/Drink.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+export const Drink = ({ drink, setDrink }) => {
+  const handleDrinkChange = (event) => {
+    setDrink(event.target.value);
+  }
+
+  return (
+    <form>
+      <div className="drinkComponent">
+        <h2>What drink would you like as a reward for surviving another day of coding?</h2>
+
+        <select
+          onChange={handleDrinkChange}
+          className="selectDropdown"
+          aria-label="Select drink"
```

Comment:
Nice! I haven't seen "aria-label" before. I guess this is just a way of "hiding" the label (so that it is not written out above the drop-down but read aloud by the SR?). I'm wondering (I do not have the answer to this) if the options also require aria-labels? But on the other hand they are read aloud by the SR as it is, so I'm guessing there is no need.